### DISCUSSION TITLE
Hunt for Hacktoberfest issues!

### DISCUSSION THREAD

**Discussion by vinitshahdeo** [vinitshahdeo/hacktoberfest-issue-hunt#3] [Announcements]
Hello 👋 

Hope you're celebrating open-source October well. 🎉  **One of the biggest challenges for beginners during [Hacktoberfest](https://hacktoberfest.digitalocean.com/) is finding issues to contribute**. I along with the [amazing community members](https://github.com/vinitshahdeo/Hacktoberfest2021/blob/main/MENTORS.md), started an [initiative](https://vinitshahdeo.dev/hacktoberfest-2021)  to help them by compiling a [handpicked list](https://github.com/vinitshahdeo/Hacktoberfest2021/blob/main/explore/ISSUES.md) of **beginner-friendly issues**. 

Being software engineers, we should always think of automation in the things we do. With this spirit, I have released an [NPM module](https://github.com/vinitshahdeo/hacktoberfest-issue-hunt) that can list `hacktoberfest` issues based on labels and languages. 

With this module, we can

- build a React App to list issues that are filtered based on languages like `JavaScript`, `Python`, etc and labels like `beginner friendly`, `first timers only`, etc.

- create a CLI tool to list the issues based on filters (languages, labels)

- author a GitHub Action to auto-update the README.md with the list of issues for Hacktoberfest.
> Having such GitHub Action will save our efforts manual efforts merging hundreds of pull requests to add issues to [`ISSUES.md`](https://github.com/vinitshahdeo/Hacktoberfest2021/blob/main/explore/ISSUES.md) in Hacktoberfest2021 repo.

Please refer [`IDEAS.md`](https://github.com/vinitshahdeo/hacktoberfest-issue-hunt/blob/main/IDEAS.md) to explore more ideas. Your innovative ideas are most welcome! 💡 

- Check out the GitHub repo [here](https://github.com/vinitshahdeo/hacktoberfest-issue-hunt).
- Check [`hacktoberfest-issue-hunt`](https://www.npmjs.com/package/hacktoberfest-issue-hunt) on NPM registry.

[![NPM](https://nodei.co/npm/hacktoberfest-issue-hunt.png?compact=true)](https://nodei.co/npm/hacktoberfest-issue-hunt/)

Kindly tweet your submissions(open-sourced repository) at [@Vinit_Shahdeo](https://twitter.com/Vinit_Shahdeo). I'd love to go through what you'll build. Some surprise goodies for the best hacks! 💪.

[![Twitter Follow](https://img.shields.io/twitter/follow/Vinit_Shahdeo.svg?style=social)](https://twitter.com/Vinit_Shahdeo)

**Happy Hacktoberfest!**

**Comment by san-coding** [comment]
Can I create a GitHub action ? For which repo?

**Reply by vinitshahdeo** [reply]
You can write an action for this repo if you wish otherwise your personal repo also works.

**Reply by san-coding** [reply]
Which action should I write ? any suggestions ?

**Reply by san-coding** [reply]
I have already setup actions for my personal repo, we have setup deploy on PR , deploy on merge and PR greeter actions, we are currently working on setting up PHP composer workflow, which GitHub action should I setup for this repo ?

**Reply by vinitshahdeo** [reply]
@san-coding A GitHub Action to auto-update the `README.md` with the list of issues for Hacktoberfest.

**Reply by san-coding** [reply]
Sure, I will soon build it

**Comment by Gauravmauryaios** [comment]
Can I work on this issue??

**Reply by vinitshahdeo** [reply]
Sure, please go ahead.
Will you be building React App?

**Comment by atharvashirdhankar** [comment]
> Hello 👋
> 
> Hope you're celebrating open-source October well. 🎉 **One of the biggest challenges for beginners during [Hacktoberfest](https://hacktoberfest.digitalocean.com/) is finding issues to contribute**. I along with the [amazing community members](https://github.com/vinitshahdeo/Hacktoberfest2021/blob/main/MENTORS.md), started an [initiative](https://vinitshahdeo.dev/hacktoberfest-2021) to help them by compiling a [handpicked list](https://github.com/vinitshahdeo/Hacktoberfest2021/blob/main/explore/ISSUES.md) of **beginner-friendly issues**.
> 
> Being software engineers, we should always think of automation in the things we do. With this spirit, I have released an [NPM module](https://github.com/vinitshahdeo/hacktoberfest-issue-hunt) that can list `hacktoberfest` issues based on labels and languages.
> 
> With this module, we can
> 
> * build a React App to list issues that are filtered based on languages like `JavaScript`, `Python`, etc and labels like `beginner friendly`, `first timers only`, etc.
> * create a CLI tool to list the issues based on filters (languages, labels)
> * author a GitHub Action to auto-update the README.md with the list of issues for Hacktoberfest.
> 
> > Having such GitHub Action will save our efforts manual efforts merging hundreds of pull requests to add issues to [`ISSUES.md`](https://github.com/vinitshahdeo/Hacktoberfest2021/blob/main/explore/ISSUES.md) in Hacktoberfest2021 repo.
> 
> Please refer [`IDEAS.md`](https://github.com/vinitshahdeo/hacktoberfest-issue-hunt/blob/main/IDEAS.md) to explore more ideas. Your innovative ideas are most welcome! 💡
> 
> * Check out the GitHub repo [here](https://github.com/vinitshahdeo/hacktoberfest-issue-hunt).
> * Check [`hacktoberfest-issue-hunt`](https://www.npmjs.com/package/hacktoberfest-issue-hunt) on NPM registry.
> 
> [![NPM](https://camo.githubusercontent.com/71a2df10683e8e7628ba17232007564ff28113f6cafea36cb6dd78ac82640de8/68747470733a2f2f6e6f6465692e636f2f6e706d2f6861636b746f626572666573742d69737375652d68756e742e706e673f636f6d706163743d74727565)](https://nodei.co/npm/hacktoberfest-issue-hunt/)
> 
> Kindly tweet your submissions(open-sourced repository) at [@Vinit_Shahdeo](https://twitter.com/Vinit_Shahdeo). I'd love to go through what you'll build. Some surprise goodies for the best hacks! 💪.
> 
> [![Twitter Follow](https://camo.githubusercontent.com/58bb098ae279fe507bb0840654ce2e89006af792976de30cba2b85c4f881d2fa/68747470733a2f2f696d672e736869656c64732e696f2f747769747465722f666f6c6c6f772f56696e69745f5368616864656f2e7376673f7374796c653d736f6369616c)](https://twitter.com/Vinit_Shahdeo)
> 
> **Happy Hacktoberfest!**

Amazing package 🚀

**Comment by mohitahlawat2001** [comment]
hi , can tell how i can contribute , i m new to open source
